src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2083,17 +2087,20 @@ void ApplicationManagerImpl::OnServiceStartedCallback(
   SDL_LOG_DEBUG("ServiceType = " << type << ". Session = " << std::hex
                                  << session_key);
   std::vector<std::string> empty;
+  std::string reason;
 
   if (kRpc == type) {
     SDL_LOG_DEBUG("RPC service is about to be started.");
-    connection_handler().NotifyServiceStartedResult(session_key, true, empty);
+    connection_handler().NotifyServiceStartedResult(
+        session_key, true, empty, reason);
     return;
   }
   ApplicationSharedPtr app = application(session_key);
   if (!app) {
     SDL_LOG_WARN("The application with id:" << session_key
                                             << " doesn't exists.");
-    connection_handler().NotifyServiceStartedResult(session_key, false, empty);
+    connection_handler().NotifyServiceStartedResult(
+        session_key, false, empty, reason);
```

Comment:
Shouldn't a reason be provided in this case?

Reply:
We've investigated this case and found that in fact there is no real case when this NotifyServiceStartedResult call may be invoked. That's why we kindly propose to delete this function call and leave here only debug message for developers.
Also corresponding UT may be deleted, because it doesn't check anything except this function call.

Proposed changes are done in 1c6840d4d5edcef26befb640c2dcc57a9c2fa20c